app/views/videos/show.html.haml

```diff
@@ -7,4 +7,35 @@
         %header
           %h3= @video.title
           %span Rating: 4.5/5.0
-        %p= @video.description
\ No newline at end of file
+        %p= @video.description
+
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map {|number| [pluralize(number, "Star")]}), class: "form-control"
```

Comment:
I would suggest to move the options to a helper method. You will reuse that logic for sure.